sdk/storage/Azure.Storage.sln

```diff
@@ -17,10 +17,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.Files.Shares"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.Files.Shares.Tests", "Azure.Storage.Files.Shares\tests\Azure.Storage.Files.Shares.Tests.csproj", "{05EBD54A-0437-46C3-A6FF-E49EC89EA22C}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Storage.Files.Shares.Perf", "Azure.Storage.Files.Shares\perf\Azure.Storage.Files.Shares.Perf\Azure.Storage.Files.Shares.Perf.csproj", "{89578DF8-BF19-46BE-BEC9-48E76C216EC9}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Storage.File.Perf", "Azure.Storage.Files.Shares\perf\Microsoft.Azure.Storage.File.Perf\Microsoft.Azure.Storage.File.Perf.csproj", "{66294703-8530-4A97-BC4A-F106DA6525EC}"
```

Comment:
I'm not sure that the performance tests are mainstream enough that it warrants loading them into the SDK-level solution.    Does it make sense to, maybe, do another filter solution for "just the stuff you need to work on the SDK"?

Reply:
Yes, that's the direction I think we should take under this directory (and possibly others over time).
'Azure.Storage.sln' can be the container for all storage related projects (and there was a request in the last PR to put them in here and remove their own solution files).
But, the sub dirs that get focused usage (like Perf, or SDK can get their separate filtered views).
That way the solution related info (overall structure, configuration etc) does not get stale across different files and all filters use one backing solution file as the true source of info.

Reply:
My concern is that this approach breaks down when you've got a service directory that contains track one code owned by one team, track two code owned by us, and multiple versions of the management libraries owned by other teams.

I've no objection to using it here, but I don't know that I agree it's the right approach universally.

Reply:
Agree. We can use it where the people working on that part of the code actually find it useful. I won't go around creating filter files all over the repo. :)